operator/builtin/input/windows/bookmark.go

```diff
@@ -0,0 +1,91 @@
+// +build windows
+
+package windows
+
+import (
+	"fmt"
+	"syscall"
+)
+
+// Bookmark is a windows event bookmark.
+type Bookmark struct {
+	handle uintptr
+}
+
+// Open will open the bookmark handle using the supplied xml.
+func (b *Bookmark) Open(offsetXML string) error {
+	if b.handle != 0 {
+		return fmt.Errorf("bookmark handle is already open")
+	}
+
+	if offsetXML == "" {
```

Comment:
Is this a silent failure?

Reply:
No. It was more to accommodate when the offsets database did not have a record on hand (which would typically happen on the first startup). Looking at this more closely though, this seems like responsibility creep. Instead, the plugin should probably check if the offset exists before calling Open.